.gitignore

```
@@ -36,4 +36,12 @@ yarn-error.log*
 next-env.d.ts
 firebase-debug.log
 firestore-debug.log
-*.log
\ No newline at end of file
+*.log
+
+#PWA Files
+**/public/sw.js
+**/public/sw.js.map
+**/public/workbox-*.js
+**/public/workbox-*.js.map
+**/public/worker-*.js
+**/public/worker-*.js.map
```

From https://github.com/vercel/next.js/tree/canary/examples/progressive-web-app:

```suggestion
#PWA Files
/public/precache.*.*.js
/public/sw.js
/public/workbox-*.js
/public/worker-*.js
/public/fallback-*.js
/public/precache.*.*.js.map
/public/sw.js.map
/public/workbox-*.js.map
/public/worker-*.js.map
/public/fallback-*.js
```